packages/browser/src/index.ts

```diff
@@ -86,6 +86,47 @@ export default class LogtoClient {
     sessionStorage.setItem(this.logtoSignInSessionKey, jsonItem);
   }
 
+  public isAuthenticated() {
+    return Boolean(this.idToken);
+  }
+
+  public getAccessToken(resource = '', scopes = []): Optional<string> {
+    const accessTokenKey = buildAccessTokenKey(resource, scopes);
+    const accessToken = this.accessTokenMap.get(accessTokenKey);
+
+    if (!accessToken) {
+      throw new LogtoClientError('access_token.failed');
+    }
+
+    if (accessToken.expiresAt < Date.now()) {
+      this.accessTokenMap.delete(accessTokenKey);
+
+      throw new LogtoClientError('access_token.expired');
```

Comment:
refresh logic missing?

Reply:
Good call, thanks. I completely dropped that part. Code updated just now, please review again.